.ci/docker/centos-rocm/Dockerfile

```diff
@@ -68,11 +77,10 @@ COPY ./common/install_rocm.sh install_rocm.sh
 RUN bash ./install_rocm.sh
 RUN rm install_rocm.sh
 COPY ./common/install_rocm_magma.sh install_rocm_magma.sh
-RUN bash ./install_rocm_magma.sh ${ROCM_VERSION}
+RUN bash ./install_rocm_magma.sh
 RUN rm install_rocm_magma.sh
-COPY ./common/install_amdsmi.sh install_amdsmi.sh
-RUN bash ./install_amdsmi.sh
-RUN rm install_amdsmi.sh
+
+ENV ROCM_PATH /opt/rocm
```

Comment:
Dont know why this is added only in centOS dockerfiles, dont remember the history. Try build by removing it.

Reply:
@pruthvistony It's not only in CentOS Dockerfile, it was actually ported from Ubuntu Dockerfile: https://github.com/pytorch/pytorch/blob/47d34261e06e2416e7a1e7d51a3d428e4ea51f9d/.ci/docker/ubuntu-rocm/Dockerfile#L63